MAUI/Carousel-View/Populating-Data.md

```diff
@@ -0,0 +1,275 @@
+---
+layout : post
+title: Populating Data in .NET MAUI Carousel View control | Syncfusion
+description: Learn here all about Populating Data support in Syncfusion .NET MAUI Carousel View (SfCarousel) control and more.
+platform : maui
+control : Carousel
+documentation : ug
+---
+
+# Populating Data in .NET MAUI Carousel View (SfCarousel)
+
+`SfCarousel` control, supports binding to different data sources such as IList Data Source, Observable Collection Data Source.
+
+## Through Binding
+
+Items can be populated in `SfCarousel` control through data source and applying custom template as explained below. 
+
+### Create a Model with Data
+
+`SfCarousel` items can be populated with a collection of image data. For example, a user may wants to create a `SfCarousel` control which will display a list of images.
+
+The `SfCarousel` model looks as follows.
+
+{% highlight C# %}
+
+namespace CarouselSample
+{
+    public class CarouselModel
+    {
+        public CarouselModel(string imageString)
+        {
+            Image = imageString;
+        }
+        private string _image;
+
+        public string Image
+        {
+            get { return _image; }
+            set { _image = value; }
+        }
+    }
+}
+
+{% endhighlight %}
+
+Populate carousel items collection in View model with the image data. 
+
+{% highlight C# %}
+
+using System.Collections.Generic;
+
+namespace CarouselSample
+{
+    public class CarouselViewModel
+    {
+        public CarouselViewModel()
+        {
+            ImageCollection.Add(new CarouselModel("carousel_person1.png"));
+            ImageCollection.Add(new CarouselModel("carousel_person2.png"));
+            ImageCollection.Add(new CarouselModel("carousel_person3.png"));
+            ImageCollection.Add(new CarouselModel("carousel_person4.png"));
+            ImageCollection.Add(new CarouselModel("carousel_person5.png"));
+        }
+        private List<CarouselModel> imageCollection = new List<CarouselModel>();
+        public List<CarouselModel> ImageCollection
+        {
+            get { return imageCollection; }
+            set { imageCollection = value; }
+        }
+    }
+}
+
+{% endhighlight %}
+
+N> Images can also be referred in PCL and from website URL as `instructed`
```

Comment:
N> Images can also be referred to in PCL and from the website URL as `instructed.`

Reply:
Addressed the changes.